tests/_utils/test_bisect.py

```diff
@@ -6,44 +9,77 @@
 from pfhedge._utils.bisect import bisect
 
 
-def test_bisect():
+def test_bisect(device: Optional[Union[str, torch.device]] = "cpu"):
```

Comment:
今回に関しては他の使い方をする見込みはないので以下で十分だと思います＆可読性を考えるとstr単体が良いと思います。他も全部まとめて修正してもらえれば特にほかは大丈夫だと思います。

```suggestion
def test_bisect(device: str = "cpu"):
```